command/validate/validate.go

```diff
@@ -18,12 +18,17 @@ func New(ui cli.Ui) *cmd {
 type cmd struct {
 	UI    cli.Ui
 	flags *flag.FlagSet
-	quiet bool
-	help  string
+	// ConfigFormat forces all config files to be interpreted as this
+	// format independent of their extension.
+	configFormat string
+	quiet        bool
+	help         string
 }
 
 func (c *cmd) init() {
 	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
+	c.flags.StringVar(&c.configFormat, "config-format", "",
+		"Config files are in this format irrespective of their extension. Must be 'hcl' or 'json'")
```

Comment:
Heh fair enough. I guess the real issue is that the other flag here should probably be reworded to be more consistent with the other commands' flag descriptions!

I'm fine either way, this was a nitpick and it's not making anything tons worse probably. I'd suggest take a super quick look at other commands and try to make sure this one is no worse after your change in terms of how the help reads maybe?

Reply:
What do you think about "Performs validation in quiet mode. This produces no output when successful" for the quiet flag help message?

Reply:
Actually I dont care for that much either. No other commands have help messages start with something to the effect of "if the flag is set", "when given" or "when set".

Maybe a better usage would be "Causes validation to prevent output when successful"